CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+**2.6.0** (5 Aug 2015)
+- add `time_event` method
+- update snippet to load mixpanel correctly from cdn when ﬁle is opened locally
```

Comment:
Oh, it's never the end of the world. It tends to only matter in insane corner cases, like "oops, my editor replaced U+2009 with U+0020 and broke that test" or "oops, Phabricator threw out half of my diff because MySQL truncated the whole thing at the first U+1F600" — i.e., it's things that all reasonable people would agree don't belong in source code. Very occasionally it matters when "oops, I opened my unit test in Word and suddenly `assertEqual(output, "ﬁle not found")` is failing", in which case nothing *super* insane is going on (except someone using Word, of course) but the git hook would still catch the mistake.

I'd definitely agree that .htm/.html files get a pass.